source/fundamentals/crud/read-operations/cursor.txt

```diff
@@ -5,4 +5,90 @@ Access Data From a Cursor
 .. default-domain:: mongodb
 
 Read operations that return multiple documents do not immediately return
-all values matching the query.
+all values matching the query. Because a query can potentially match
+large sets of documents, these operations rely upon an object called a
+:java-docs:`MongoCursor </apidocs/mongodb-driver-sync/com/mongodb/client/MongoCursor.html>`.
+
+A ``MongoCursor`` fetches documents in batches to reduce both memory
+consumption and network bandwidth usage. 
+
+The following functions use cursors internally, and return the results of the operations:
+
+- ``collection.find()``
+- ``collection.aggregate()``
+- ``collection.listIndexes()``
+- ``db.listCollectionNames()``
+
+Append ``.cursor()`` to use these functions as cursors.  
+
+The following shows ``collection.find()`` as a cursor:
+
+.. code-block:: java
+   :copyable: true
+   
+   collection.find().cursor();
+
+Cursor Paradigms
+----------------
+
+You work with cursors using **cursor paradigms**. Most cursor paradigms
+allow you to access query results one document at a time, abstracting
+away network and caching logic. 
+
+.. warning::
+
+   Do not combine different cursor paradigms on a single cursor.
+   Operations such as ``next()`` and ``forEachRemaining()``
+   each predictably modify the original cursor. If you mix these calls
+   on a single cursor, you may receive unexpected results.
```

Comment:
Just tested it and it is possible, but nothing out of the ordinary happens.  It prints all documents as expected and throws no error when the functional iteration runs.  I'll remove this since it doesn't apply here either.